test/suite/test_cursor_bound05.py

```diff
@@ -37,7 +37,7 @@ class test_cursor_bound05(bound_base):
     file_name = 'test_cursor_bound05'
     key_format = 'S'
     start_key = 1000
-    end_key = 2000
+    end_key = 1999
```

Comment:
This deserves a comment explaining why it is correct, I understand the issue but it does look odd at first glance.